source/devices/AM335X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -44,19 +44,21 @@ Documentation
    running the demonstration application that is loaded on flash. This
    document is provided as part of the EVM kit.
 
-Release 09.01.00
-================
+Release 09.03.05.02
+===================
 
-Released Oct 2023
+Released Dec 2024
 
 .. rubric:: What's New
    :name: whats-new
 
-.. note:: Generic PRU-ICSS Ethernet is descoped in 09.01.00 release.
+.. note:: Generic PRU-ICSS Ethernet is descoped in 09.03 release.
 
-Processor SDK 9.1 Release has following new features:
+Processor SDK 9.3 Release has following new features:
 
- - 2023 LTS Refresh with Stable Kernel 6.1, U-boot 2023.04, Yocto Kirkstone 4.0
+ - 2023 LTS Stable Update to 6.1.119
+ - Splash Screen Feature
+ - Important Bug Fixes
```

Comment:
Is there a plan to link this section to Issue Tracker section?

Reply:
Issue tracker will update separately in a different PR